packages/wdio-cli/src/interface.js

```diff
@@ -34,9 +34,11 @@ export default class WDIOCLInterface extends EventEmitter {
         this.clockTimer = 0
         this.jobs = new Map()
         this.start = Date.now()
+        // totalWorkerCnt - retries = finished = passed + failed
```

Comment:
should this stay commented out?

Reply:
This is actually a comment - I'll clarify a bit better.